.circleci/config.yml

```diff
@@ -47,6 +60,10 @@ jobs:
             # Maven heap size
             MAVEN_OPTS: -Xmx256m -Xms256m
           command: |
+            kubectl -n flyte port-forward deployment/flyteadmin 8089:8089 &
+            # it takes some time for the portforwarding to be ready
```

Comment:
Do a spin check with timeout here to make sure it starts instead of guess-sleep?

Reply:
Yes. In that case we can install `grpc_health_probe` to make sure it starts.

Reply:
We don't restart/redeploy flyte every time we run e2e test, right? The spin check I mentioned here is to make sure port forwarding is ready. But using grpc_health_probe is more reliable.